dev/tasks/docker-tests/github.linux.yml

```diff
@@ -45,21 +45,29 @@ jobs:
             {{ image }} \
             {{ command|default("") }}
 
-    {% if '-r' in image %}
       - name: Dump R install logs
-        run: cat arrow/r/check/arrow.Rcheck/00install.out
-        continue-on-error: true
         if: always()
+        run: |
+          log=arrow/r/check/arrow.Rcheck/00install.out
+          if [ -f ${log} ]; then
+            cat ${log}
+          fi
       - name: Dump R test logs
-        run: cat arrow/r/check/arrow.Rcheck/tests/testthat.Rout*
         if: always()
+        run: |
+          for log in arrow/r/check/arrow.Rcheck/tests/testthat.Rout*; do
+            if [ -f ${log} ]; then
+              echo ${log}
```

Comment:
Do we need this echo line?

Reply:
It will help us to distinct file content when there are multiple log files.
But if R developers don't need it, I can remove this.

Reply:
Aaah oops I forgot about the `*`! You’re right, let’s keep it in